src/util/format.ts

```diff
@@ -28,3 +28,29 @@ export function convertToString(unit: Unit, num: number | null): string {
   }
   return num.toFixed(8).toString();
 }
+
+export function formatAmount(value: string, unit: Unit): string {
```

Comment:
Should this have a test or is it covered by the other updated tests?

Reply:
Will be done in #306 :+1: